cmd/update/update.go

```diff
@@ -0,0 +1,23 @@
+// maestro-cli
+// https://github.com/topfreegames/maestro-cli
+//
+// Licensed under the MIT license:
+// http://www.opensource.org/licenses/mit-license
+// Copyright © 2017 Top Free Games <backend@tfgco.com>
+
+package update
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// Cmd represents the create command
+var Cmd = &cobra.Command{
+	Use:   "update",
+	Short: "Updates a scheduler",
```

Comment:
@caiordjesus this operator description should be generic, we can remove the reference to the "scheduler" gere

Reply:
Awesome!